internal/commands/loglevel/set.go

```diff
@@ -62,22 +62,24 @@ func updateDebuggingConfiguration(ctx context.Context, logSeverity libsveltosv1a
 		)
 	}
 
-	return updateLogLevelConfiguration(ctx, spec)
+	return updateLogLevelConfiguration(ctx, spec, namespace, clusterName)
 }
 
 // Set displays/changes log verbosity for a given component
 func Set(ctx context.Context, args []string) error {
 	doc := `Usage:
-  sveltosctl log-level set --component=<name> (--info|--debug|--verbose)
+  sveltosctl log-level set --component=<name> --namespace=<namespace> --cluster=<cluster-name> (--info|--debug|--verbose)
```

Comment:
namespace and cluster are optional (and cluster-type as well).

So those need to be 

[--namespace=<namespace>] [--cluster=<cluster-name>]